synthesizer/program/src/logic/command/get_or_use.rs

```diff
@@ -31,10 +31,8 @@ use console::{
 pub struct GetOrUse<N: Network> {
     /// The mapping.
     mapping: CallOperator<N>,
-    /// The key to access the mapping.
-    key: Operand<N>,
-    /// The default value.
-    default: Operand<N>,
+    /// The operands.
+    operands: [Operand<N>; 2],
```

Comment:
Why the change here? It felt more clear with the direct variables `key` and `default`.

At the very least, the documentation of `operands` should reflect the structure.

Reply:
Ditto with the other commands that have this unification

Reply:
This was needed to support the `operands()` on commands without allocating `Vec`s
The instructions are actually implemented in a similar way.
TBH the implementations of both instructions and commands needs some refactoring to standardize.

Reply:
For additional context, we need to iterate through all of the operands in a program and check that they don't match `Operand::Checksum` or `Operand::Edition` before a specific consensus version. It would be ideal if we can do this without cloning the `Operand`s